### ISSUE TITLE
[SUGGESTION] Config Option to Allow TARDIS Travel Only to Previously Visited Dimensions

### ISSUE THREAD

**Issue by Qanthelas** [DarkholmeTenk/TardisModIssues#300]
tl;dr A config option (True or False) to toggle if the TARDIS can take a player to a dimension they have never been to before.

Many mods put a lot of effort into making the player do something difficult to reach a certain dimension, and I feel like the TARDIS could be used to bipass that.  Rather than blacklisting that dimension, I was thinking that maybe a simple and straightforward solution would be an option to only allow TARDIS travel to a dimension after it has been visited first by some other means.  This means the Dimension selection part of the TARDIS console would start out with just the starting dimension (generally the Overworld) and would add each new dimension as it is visited (the Nether, the End, etc.)  This way I could allow TARDIS travel to every dimension once it is discovered first, but disallow people bypassing standard progression (like getting Obsidian to make a Nether Portal.)

As a big picture idea, I want to be able to give a player a TARDIS early on in a quest-driven modpack without breaking the whole progression of the pack.

DISCLAIMER: I have not tried the TARDIS mod in any modpack that allows the player to have a TARDIS early on, maybe this feature already exists somehow.

**Comment by DarkholmeTenk** [comment]
#192 #191

**Comment by handicraftsman** [comment]
Yes, it works.
